janus_core/processing/correlator.py

```diff
@@ -179,10 +179,12 @@ class Correlation:
 
     Parameters
     ----------
-    a : tuple[Observable, dict]
-        Getter for a and kwargs.
-    b : tuple[Observable, dict]
-        Getter for b and kwargs.
+    n_atoms : int
+        Number of possible atoms to track.
```

Comment:
Sorry, I think we've discussed this a few times already, but does this only refer to the number of atoms being tracked when you want to store a quantity for each of those atoms, as opposed to an observable that could depend on `n_atoms`, but we're only interested in a single value for it?

I assume they're quite different, but to me the description could refer to either?

Reply:
I don't think it is required actually, since now it is all inferred. Must have got left or re-added on a re-base I will check